tests/swift/Wasm.tests/Tests/FlatBuffers.Test.Swift.WasmTests/monster_test_generated.swift

```diff
@@ -301,7 +301,7 @@ public struct MyGame_Example_Test_Mutable: FlatBufferObject {
   @discardableResult public func mutate(a: Int16) -> Bool { return _accessor.mutate(a, index: 0) }
   public var b: Int8 { return _accessor.readBuffer(of: Int8.self, at: 2) }
   @discardableResult public func mutate(b: Int8) -> Bool { return _accessor.mutate(b, index: 2) }
-  
+
```

Comment:
I don't remember what caused this file to be touched (as noted elsewhere, I try not to touch files because my editors like wiping out whitespace...), but this change was undone by https://github.com/google/flatbuffers/pull/8177/commits/29f08c92fc507180126f9aa58c353127f27e2c6a